frontend/lib/src/components/core/Block/styled-components.ts

```diff
@@ -136,8 +136,7 @@ export const StyledVerticalBlock = styled.div<StyledVerticalBlockProps>(
   ({ width, theme }) => ({
     width,
     position: "relative", // Required for the automatic width computation.
-
-    display: "flex",
+    display: width === 0 ? "none" : "flex",
```

Comment:
I think we might not even need it. But I think in any case, if the width of a vertical container is 0, we should better not display the container.

Reply:
I'm lukewarm to this, I guess the issues that could pop up is "my content is not displayed" vs "my content flickers or is mislayout". I think it's probaly good enough.

Reply:
Good point. Yeah, I think it could make some of the very edge cases harder to detect. I have removed this again. If we see some related flickering in the future, we could decide to add this again.